lib/utils.js

```diff
@@ -65,7 +65,11 @@ const getBabelOptions = function loadBabelOptions(filename, options = {}) {
 }
 
 const getTsJestConfig = function getTsJestConfig(config) {
-  const isUsingTs = resolveTsConfigSync(process.cwd())
+  const tsConfigPath = path.resolve(
+    process.cwd(),
+    config.globals?.['vue-jest']?.tsConfig ?? ''
```

Comment:
[Optional Chaining](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Optional_chaining) is not supported on Node < 14.
```suggestion
    getVueJestConfig(config).tsConfig || ""
```